shotover-proxy/example-configs/cassandra-request-throttling/topology.yaml

```diff
@@ -0,0 +1,15 @@
+---
+sources:
+  cassandra_prod:
+    Cassandra:
+      listen_addr: "127.0.0.1:9042"
+chain_config:
+  main_chain:
+    - RequestThrottling:
+        max_requests_per_second: 40000
```

Comment:
Hmmmmm...
I think the only way to get consistent results from this benchmark would be to always allow messages through (some absurdly high number) or never allow messages through (value of 1 is close enough)
Otherwise we'll get all sorts of weird performance implications from reaching or not reaching the max on different machines and at different points of the 3 second benchmark.

Reply:
Yes, I thought 40k would be enough to allow all through.

Reply:
ah, lets add a comment explaining that and maybe bump it up a bit more, should be safe at 100,000,000

Reply:
Should we also have a second benchmark that lets 1 message through?

Reply:
I don't think 1 would work because the driver sends a bunch of startup messages. 

I'm starting to think we should set a minimum allowed value for the request throttling because it can break things if you set it too low.

Reply:
> I don't think 1 would work because the driver sends a bunch of startup messages.

alright, the single bench we have now should be enough.

> I'm starting to think we should set a minimum allowed value for the request throttling because it can break things if you set it too low.

That makes sense to me.
Picking an appropriate value could be tricky though.
We need plenty of buffer in case the handshake changes between cassandra releases.

Reply:
After a bit of testing, the lowest value I could get to work was 15, so I think 20 is a reasonable minimum.

Reply:
I would go to 50.
I think its better to be on the safe side here.